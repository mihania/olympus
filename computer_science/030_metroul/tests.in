11
4 9 1 8
1 2 3 4 5
2 6 8
7 6 4
5 9
12 13 1 13
1 2
2 3
3 4
4 5
5 6
6 7
7 8
8 9
9 10
10 11
11 12
12 13
12 13 1 13
6 7
1 2
7 8
2 3
9 10
3 4
11 12
4 5
10 11
5 6
8 9
12 13
12 241 1 13
1 2 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34
2 3 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55
3 4 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76
4 5 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97
5 6 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115
6 7 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137
7 8 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155
8 9 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173
9 10 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190
10 11 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207
11 12 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224
12 13 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241
12 241 1 13
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
12 241 2 13
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
12 241 3 13
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
12 241 4 13
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
12 241 4 10
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
13 243 1 13
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2 242
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13 243
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
242 243
13 243 1 13
1 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 2 242
6 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 7
2 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 3
7 138 139 140 141 142 143 144 145 146 147 148 149 150 151 151 153 154 155 8
3 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 4
9 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 10
4 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 5
10 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 11
12 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 13 243
5 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 6
8 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 9
11 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 12
242 244 245 243
