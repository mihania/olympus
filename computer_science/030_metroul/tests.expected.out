1
11
11
11
11
10
9
8
5
2
2
